### ISSUE TITLE
Accessibility Problem [Unlabeled]

### ISSUE THREAD

**Issue by mablue** [hiddify/hiddify-next#557]
### Search first

- [X] I searched and no similar issues were found

### What Happened?

https://github.com/hiddify/hiddify-next/assets/5004900/dfdf9e12-330d-45bd-b1c1-3f1194bd9542

### Reproduce the Bug

Icons are Unlabeled and blinds cant work with it.

### Version

Dev

### Platform/OS

Android

### Are you willing to submit a PR? If you know how to fix the bug.

- [X] I'm willing to submit a PR (Thank you!)

**Comment by problematicconsumer** [comment]
Unfortunately, I'm not able to play the video. Can you give more details?

**Comment by mablue** [comment]
Yes. Please open TalkBack in your phone. And explore with your finger as you here on some buttons and places the phone TTS can't find a label in it and generates this sound: unlabeled.
Its not user finendely for a blind person that want to know where is its finger
